include/swift/AST/Ownership.h

```diff
@@ -142,6 +142,19 @@ enum class ValueOwnership : uint8_t {
 enum : unsigned { NumValueOwnershipBits =
   countBitsUsed(static_cast<unsigned>(ValueOwnership::Last_Kind)) };
 
+static inline llvm::StringRef getOwnershipSpelling(ValueOwnership ownership) {
+  switch (ownership) {
+  case ValueOwnership::Default:
+    return "";
```

Comment:
Should this be something more concrete i.e. "default"?

Reply:
Leaving the spelling as is, because it matched both swift and sil.